pkgs/development/ocaml-modules/synchronizer/default.nix

```
@@ -0,0 +1,32 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildDunePackage,
+  prelude,
+}:
+
+buildDunePackage rec {
+  pname = "synchronizer";
+  version = "0.1";
+
+  minimalOCamlVersion = "4.14";
+
+  src = fetchFromGitHub {
+    owner = "OCamlPro";
+    repo = pname;
```

```suggestion
    repo = "synchronizer";
```
do not rely on pname and repo name matching, specify it explicitly